shovels_status.go

```diff
@@ -0,0 +1,37 @@
+package rabbithole
+
+import (
+	"net/url"
+)
+
+// ShovelStatus contains the status of a shovel
+type ShovelStatus struct {
+	// Shovel name
+	Name string `json:"name"`
+	// Virtual host this shovel belongs to
+	Vhost string `json:"vhost"`
+	// Type of this shovel
+	Type string `json:"type"`
+	// State of this shovel
+	State string `json:"state"`
+	// Timestamp of this shovel
```

Comment:
This timestamp is of the time when the Shovel was started. We should either clarify this or remove this comment altogether.